dockerfile_parse/parser.py

```diff
@@ -330,40 +333,20 @@ def _instruction_getter(self, name, env_replace):
             raise ValueError("Unsupported instruction '%s'", name)
         instructions = {}
         envs = self.parent_env.copy()
-        for insndesc in self.structure:
-            this_insn = insndesc['instruction']
-            if this_insn in (name, 'ENV'):
-                logger.debug("%s value: %r", name.lower(), insndesc['value'])
-                shlex_splits = shlex_split(insndesc['value'],
-                                           env_replace=env_replace, envs=envs)
-                if '=' not in shlex_splits[0]:  # LABEL/ENV name value
-                    # split it to first (name) and the rest (value)
-                    key_val = insndesc['value'].split(None, 1)
-                    key = strip_quotes(key_val[0])
-                    try:
-                        val = key_val[1]
-                    except IndexError:
-                        val = ''
-
-                    if env_replace:
-                        val = EnvSubst(val, envs).substitute()
-
-                    val = remove_nonescaped_quotes(val)
-                    if this_insn == name:
-                        instructions[key] = val
-                        logger.debug("new %s %r=%r", name.lower(), key, val)
-                    if env_replace and this_insn == 'ENV':
```

Comment:
The `env_replace` check seems to have been removed. This is probably fine, especially since this was only being done for this instruction format, and not when `"="` was used.